app/experimenter/nimbus-ui/src/hooks/useCommonForm.tsx

```diff
@@ -7,6 +7,38 @@ import { RegisterOptions, FieldError } from "react-hook-form";
 import Form from "react-bootstrap/Form";
 import { useForm } from "react-hook-form";
 
+// exported for testing
```

Comment:
Should get rid of these if you rebase, these were scooted into their relevant page files 🙂